study.md

```diff
@@ -46,7 +53,18 @@ other named Mr.MagicNibbles (yellow) and bites a bit. Please copy the JSON you
 already wrote and include the information about Dave's fish.
 
 ```json
-<!-- your answer here -->
+{ "Jason" : {
```

Comment:
I would have gone with an array of ojkect for the petname1, this this is still correct.